software/webapp/src/test/java/brooklyn/entity/webapp/jboss/JBossPasswordHashingTest.java

```diff
@@ -18,42 +18,44 @@
  */
 package brooklyn.entity.webapp.jboss;
 
+import static org.testng.Assert.assertEquals;
+
 import org.testng.annotations.Test;
 
-import static org.testng.Assert.assertEquals;
+import brooklyn.entity.webapp.JavaWebAppSshDriver;
 
 /**
  * Expected values in tests were generated by AS7's add-user.sh script and copied here.
  */
-public class JBoss7PasswordHashingTest {
+public class JBossPasswordHashingTest {
 
     @Test
     public void testPasswordForManagementRealm() {
         assertEquals(
-                JBoss7SshDriver.hashPassword("username", "password", "ManagementRealm"),
+        		JavaWebAppSshDriver.hashPassword("username", "password", "ManagementRealm"),
```

Comment:
Yes since writing this code I changed my Eclipse settings so this won't happen in the future.